backend/user_management/schemas.py

```
@@ -81,15 +86,15 @@ class Config:
 
 class ProductCreate(BaseModel):
     #brand_id: int  # Ensure the brand exists before creating a product
-    product_name: str
-    product_pic: List[str]  # Array of image storage links
-    product_video: List[str]  # Array of video storage links
-    category: str
-    description: Optional[str]
+    product_name: str = Field(..., min_length=3, max_length=100)
+    product_pic: List[str]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)